src/editors/containers/TextEditor/components/SelectImageModal/hooks.test.js

```diff
@@ -254,9 +276,23 @@ describe('SelectImageModal hooks', () => {
       expect(click).toHaveBeenCalled();
     });
     describe('addFile (uploadImage args)', () => {
-      const event = { target: { files: [testValue] } };
+      const eventSuccess = { target: { files: [{ value: testValue, size: 2000 }] } };
+      const eventFailure = { target: { files: [testValueInvalidImage] } };
+      it('image fails to upload', () => {
```

Comment:
a more descriptive test title will help the reader "an image fails to upload _if_ ___"